### PR TITLE
Enhacement for pre-split dataset and more

### PR DESCRIPTION
**Pull request by CeVague**
More logical upload interface and feedback
Support pre-split dataset
Generate .info file
General fix and enhancement

### CODE REVIEW

File: bundler/tasks.py [hunk 1]
```diff
@@ -62,6 +62,48 @@ def gen_reference(output_dir, dataset, phase_id):
...
+                        if cont_name == 'info':
+                            filename = '%s_public.info' % name
+
+                        content.raw_content.open()
+                        copy_file_field(content.raw_content,
+                                        path.join(tmp_dir, filename))
+                        content.raw_content.close()
+
+
+def create_info_file(challenge, tmp_dir):
```
Comment (ckcollab) [inline bundler/tasks.py]: Hmm, why not use `.json` or something similar here?

File: chalab/templates/chalab/_wiki_help.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {% if uri %}
 <div class="wiki-help">
-    <a href="https://github.com/CodalabOrg/chalab/wiki/Help:-{{ uri }}"
+    <a href="https://github.com/codalab/chalab/wiki/Help:-{{ uri }}"
```
Comment (ckcollab) [inline chalab/templates/chalab/_wiki_help.html]: 👍

**Comment by ckcollab** [comment]
Can you give some instructions on how to test this? Thanks!

File: bundler/tasks.py (hunk 1 above)
Comment (CeVague) [inline bundler/tasks.py]: I have no choice for that ^^
The dataset can contain a .info file, which is defined with this "format".

**Comment by CeVague** [comment]
Hum, the biggest part to test is the first step : the upload of the dataset.

You can try with : 
- A normal dataset with data and solution (in this case you can modify the split in the next step)
- A pre-split dataset (in this case you can't modify the split)
- A dataset with other useless files (all will work but a warning alert you of the ignored files)
- A dataset with missing files (an alert\error message will appeared)

Also, you can verify that informations put in the .info file (in the bundle once build) are correct.

**Comment by ckcollab** [comment]
Where can I test this, is there maybe a staging server with this already running on it?

**Comment by CeVague** [comment]
No, sorry.
I run it locally only.

**Comment by ckcollab** [comment]
Would it be possible to change the existing server to this branch, or to setup a staging server?

**Comment by ckcollab** [comment]
Also googling "codalab chalab" I found this: 
http://staging.chalab.eu/

Do you not have access to that server?

**Comment by CeVague** [comment]
Yep, I think I can synchronise the server with this branch.

And no, this server probably belongs to Laurent (the precedent developer). So I don't think I can use it.

**Comment by dragonlocke** [comment]
Where is a place I can test this? Also, do you have any examples?

**Comment by ckcollab** [comment]
@CeVague Could you post some examples to test this with? This is the correct URL: https://chalab.lri.fr/ ?

**Pull request merged**